### DISCUSSION TITLE
Favorite tracks

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#81] [Q&A] [answered]
Cruiser now remembers the imported tracks (and their colors), with no way to manage them.
Like show / hide some of them, change color, etc. Need to clear overlays and re-import them.

Maybe we need a way to manage the tracks, like the favorite routes:
- https://github.com/devemux86/cruiser/discussions/35

**Comment by devemux86** [comment]
"**Favorite tracks**" preview:

- Select track color and visibility
- Organized in groups
- Available in search
- Import / export

<table>
<tr>
<th>Track groups</th><th>Favorite tracks</th>
</tr>
<tr>
<td><img src="https://github.com/devemux86/cruiser/assets/3484020/9b44ba1c-c24d-4ec4-9b4f-5332576abac5"/></td><td><img src="https://github.com/devemux86/cruiser/assets/3484020/d0af3f6a-548e-4928-b181-5205730b47ab"/></td>
</tr>
<tr>
<th>Favorites menu</th><th>Search tracks</th>
</tr>
<tr>
<td><img src="https://github.com/devemux86/cruiser/assets/3484020/4abefdb8-f07f-4bae-9b28-a9344ff04d66"/></td><td><img src="https://github.com/devemux86/cruiser/assets/3484020/e33cb75e-9371-4687-b6be-ebe4974731d3"/></td>
</tr>
</table>

**Reply by devemux86** [reply]
About the "track" icon, we can use:

![route](https://github.com/devemux86/cruiser/assets/3484020/72228702-50c0-41d6-a640-a8841f554414) ![timeline](https://github.com/devemux86/cruiser/assets/3484020/327f100e-2fee-4543-8b6d-62c820987a8d)

Or something else from [here](https://fonts.google.com/icons?icon.set=Material+Icons&icon.style=Filled)?

---

The "route" icon should be ok?
To remind us that routes are ready for navigation.

**Reply by airald** [reply]
I preffer timeline icon for track.

**Reply by airald** [reply]
Route is ok.

**Comment by devemux86** [comment]
Favorite tracks can be loaded as:
- **Route**: calculate a route from the track (like import dialog)
- **Track**: overlay their path with any color

Bonus: we can save the **recorded** GPS track in the favorite tracks.

**Comment by devemux86** [comment]
Favorite tracks are available for testing in:
- https://github.com/devemux86/cruiser/discussions/329

**Comment by devemux86** [comment, marked as answer]
Favorite tracks is available in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
